Add in metrics API endpoint

davidcollom opened the pull request.

davidcollom commented:
Hey,
I've spent a little longer looking at the tests and I'm not sure if I'm missing something but these tests complete successfully on my local machine?

The error from travis seems to me related to the test not being able to find a vcr for the specific call?

I'm happy to look into it further, if someone could point me in the right direction?

Cheers

felixb commented:
Tests are running fine on my machine too. I don't get it neither.
Maybe the .ping test is broken, because it's written in different syntax.

Maybe it's just flaky? You could try squashing your commits and let travis try again.

Thanks for your contributions.

davidcollom commented:
Hi @felixb,
I've updated the ping test to force vcr into naming the cartridge with an actual name.

it would appear that vcr (locally) was actually saving it as a .yml which I'm assuming travis was ignoring/removing?

I renamed the test which appears to force vcr into giving it a better name and shows as passing now.
